src/addrman_impl.h

```diff
@@ -0,0 +1,287 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_ADDRMAN_IMPL_H
+#define BITCOIN_ADDRMAN_IMPL_H
+
+#include <logging.h>
+
+#include <sync.h>
```

Comment:
Should be grouped above with logging.h, since sync.h is a project header and not a stl header.

Reply:
done